packages/react-server/src/ReactFizzServer.js

```diff
@@ -410,11 +417,40 @@ function popComponentStackInDEV(task: Task): void {
   }
 }
 
-function logRecoverableError(request: Request, error: mixed): void {
+// stash the component stack of an unwinding error until it is processed
+let lastBoundaryErrorComponentStackDev: ?string = null;
+
+function captureBoundaryErrorDetailsDev(
```

Comment:
Since this returns void anyway, just to be safe, you can wrap it all in DEV checks.